src/components/Results/DetectedIssueResources.tsx

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
 import { Accordion, AccordionSummary, AccordionDetails, Grid, Typography } from '@material-ui/core';
 import { R4 } from '@ahryman40k/ts-fhir-types';
+import { Enums } from 'fqm-execution';
 import fhirpath from 'fhirpath';
 
 interface Props {
   detectedIssue: R4.IDetectedIssue;
 }
 
 const DetectedIssueResources: React.FC<Props> = ({ detectedIssue }) => {
-  const guidanceResponseArray = fhirpath.evaluate(detectedIssue, 'contained.GuidanceResponse');
+  const guidanceResponses = fhirpath.evaluate(detectedIssue, 'contained.GuidanceResponse');
+  const guidanceResponseArray = [];
+
+  for (let i = 0; i < guidanceResponses.length; i++) {
+    const reasonCode = fhirpath.evaluate(guidanceResponses[i], 'reasonCode.coding.code')[0];
+    if (reasonCode === Enums.CareGapReasonCode.MISSING || reasonCode === Enums.CareGapReasonCode.PRESENT) {
+      guidanceResponseArray.push(guidanceResponses[i]);
+    } else {
+      guidanceResponseArray.unshift(guidanceResponses[i]);
+    }
+  }
```

Comment:
Thoughts on switching to a `forEach` here? It's more javascripty

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

code will remain the same, you just don't need the `i` index